ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -1232,16 +1232,18 @@ Status MarkForClustering(Graph* graph, const std::set<string> skip_these_nodes,
       }
 
       // Check if op is supported by backend
-      bool is_supported = false;
-      TF_RETURN_IF_ERROR(IsSupportedByBackend(node, op_backend, TFtoNgraphOpMap,
-                                              is_supported));
-
-      if (!is_supported) {
-        NGRAPH_VLOG(5) << "TF Op " << node->name() << " of type "
-                       << node->type_string()
-                       << " is not supported by backend: " << ng_backend_type;
-        break;
-      }
+      // bool is_supported = false;
+      // TF_RETURN_IF_ERROR(IsSupportedByBackend(node, op_backend,
+      // TFtoNgraphOpMap,
+      //                                         is_supported));
+
+      // if (!is_supported) {
+      //   NGRAPH_VLOG(5) << "TF Op " << node->name() << " of type "
+      //                  << node->type_string()
+      //                  << " is not supported by backend: " <<
+      //                  ng_backend_type;
+      //   break;
+      // }
```

Comment:
Remove; you don't need to comment this out.

Reply:
removed

Reply:
What I meant was - do you really need to comment this out? What about the opset3 upgrade does it make unsupported on the current backends supported on the main branch?